### PR TITLE
Rename conditional cases

### PR DESCRIPTION
**Pull request by gbalint**
**Problem:**
We started to use two distinct terminology for conditional cases: then/else and true/false.

**Fix:**
Unify the terminology by only using `true-case` and `false-case`.

### CODE REVIEW

File: editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx [hunk 1]
```diff
@@ -220,8 +220,8 @@ xdescribe('conditionals in the navigator', () => {
           regular-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2
             conditional-clause-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/then-then-div-then
               regular-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/then-then-div
-            conditional-clause-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/else-case-else
-              synthetic-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/else-case-attribute
+            conditional-clause-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/false-case-else
```
Comment (Rheeseyb) [inline editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx:223]: Hmmm why has this ended up as `false-case-else`, whereas the tests in `ui-jsx-canvas-conditionals.spec.browser2.tsx` have been changed to `false-case-false-case`?

File: editor/src/core/model/element-template-utils.spec.tsx [hunk 2]
```diff
@@ -957,17 +957,17 @@ describe('findJSXElementChildAtPath', () => {
       'utopia-storyboard-uid/scene-aaa/app-entity:aaa/parent/conditional-1/ternary-false-root/ternary-false-child',
     ])
 
-    // !!!! Do I misunderstand something?? shouldn't the then-case and else-case return the true-root and false-root here?? these tests fail now...
+    // !!!! Do I misunderstand something?? shouldn't the true-case and false-case return the true-root and false-root here?? these tests fail now...
```
Comment (Rheeseyb) [inline editor/src/core/model/element-template-utils.spec.tsx:960]: This comment seems to be invalid now...

File: editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx (hunk 1 above)
Comment (gbalint) [reply editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx:223]: I just missed it and the test is disabled, so it did not break the build.

Comment (gbalint) [reply editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx]: I reverted my changes and added a separate task to fix and enable this test

**Review by ruggi — APPROVED** [review]

**Review by bkrmendy — APPROVED** [review]

**Review by seanparsons — APPROVED** [review]

**Pull request merged**